src/order/succ_pred/interval_succ.lean

```diff
@@ -0,0 +1,56 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import order.succ_pred.basic
+import data.set.lattice
+
+/-!
+# Intervals `Ixx (f x) (f (order.succ x))`
+
+In this file we prove
+
+* `monotone.bUnion_Ico_Ioc_map_succ`: if `α` is a linear archimedean succ order and `β` is a linear
+  order, then for any monotone function `f` and `m n : α`, the union of intervals
+  `set.Ioc (f i) (f (order.succ i))`, `m ≤ i < n`, is equal to `set.Ioc (f m) (f n)`;
+
+* `monotone.pairwise_disjoint_on_Ioc_succ`: if `α` is a linear succ order, `β` is a preorder, and
+  `f : α → β` is a monotone function, then the intervals `set.Ioc (f n) (f (order.succ n))` are
+  pairwise disjoint.
+
+Later this file should be extended with lemmas about other intervals and antitone functions.
+-/
+
+open set
```

Comment:
```suggestion
open order set
```
so that you can nicely spell it `succ` and `pred`.

Reply:
Done.